Create SIP-0008.md

TravinImmunefi opened the pull request:
Relevant forum thread - https://forum.sovryn.app/t/draft-sip-0008-sovryn-bug-bounty-program-on-immunefi/171?u=travinimmunefi

YagoBit commented on SIP-0008.md:
I recommend pegging payout to USD. Additionally, Payout of up to 50% allowable in SOV (subject to vesting) at Sovryn's discretion.

TravinImmunefi replied:
Ah, sorry, I thought on the forum you meant the payout with vesting was just for the bonus. We're generally fine with vesting for large amounts, but we do recommend capping the length of them to 6 months based on feedback we've received from community members. 

I'm definitely ok with having the payouts pegged to USD and have updated it now.

TravinImmunefi commented:
I have now updated the SIP with the changes proposed by @YagoBit and have also updated the "Total Funds Requested" section to be in line with these changes.
